operator/Makefile

```diff
@@ -51,6 +51,7 @@ apply: ## Apply the controller into your ~/.kube/config cluster
 		$(HELM_OPTS) \
 		--set controller.image=ko://github.com/awslabs/kubernetes-iteration-toolkit/operator/cmd/controller \
 		--set webhook.image=ko://github.com/awslabs/kubernetes-iteration-toolkit/operator/cmd/webhook \
+		--set serviceAccount.create=true \
```

Comment:
what happens if SA already exists in the cluster and someone runs `make apply`?

Reply:
It will work if it already exists and was created by the helm chart. If it was manually created though then helm will have a conflict with it.